### DISCUSSION TITLE
Add return to top on website

### DISCUSSION THREAD

**Discussion by joaoesp** [Leaflet/Leaflet#8788] [Ideas]
Hi, I can't find a go-to top button anywhere, I think it makes a lot of sense since it's a heavy scrolling website. I believe it would be very handy.
Cheers

**Comment by IvanSanchez** [comment]
`Home` key on your keyboard.

**Reply by Falke-Design** [reply]
Doesn't work on my QWERTZ keyboard

**Reply by mourner** [reply]
On my Mac, I always use Cmd-Up (I guess there's a similar shortcut on Win/Linux).

**Comment by mourner** [comment]
If this is about the API reference page which is the only one that's long, perhaps we should make the "table of contents" at the top left clickable:
 
<img width="584" alt="image" src="https://user-images.githubusercontent.com/25395/211516644-ea5baf66-c0ad-419b-9cb7-2c16d1b6cfec.png">
